src/Subscription/ChangeLog.md

```diff
@@ -18,6 +18,13 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* All cmdlets parameter SubscriptionName has the alias DisplayName.
```

Comment:
this is not release note, this is description of your PR

Reply:
The content is updated to：
```
* Invoke-AzSubscriptionAcceptOwnership:
    1. Parameter ['-SubscriptionName'] is given an alias [-DisplayName].
* New-AzSubscriptionAlias:
    1. Since the arguments [`-SubscriptionId`] and [`-BillingScope`],[`-SubscriptionName`],[`-Workload`] are mutually exclusive, they are split into two different syntaxes.
    2. Parameter ['-SubscriptionName'] is given an alias [-DisplayName].
* Rename-AzSubscription:
    1. Parameter [`-SubscriptionName`] was required.
    2. Parameter ['-SubscriptionName'] is given an alias [-DisplayName].
* Removed cmdlet `Update-AzSubscriptionPolicy`
```